LLama/Native/NativeApi.LLava.cs

```diff
@@ -88,7 +62,7 @@ public struct llava_image_embed
     /// <param name="embed"></param>
     /// <returns></returns>
     [DllImport(llavaLibraryName, EntryPoint = "llava_image_embed_free", CallingConvention = CallingConvention.Cdecl)]
-    public static extern llava_image_embed* llava_image_embed_free(llava_image_embed* embed);
+    public static extern LLavaImageEmbed* llava_image_embed_free(LLavaImageEmbed* embed);
```

Comment:
If a `LLavaImageEmbed` is being allocated in some methods and free in another it should be handled with a `SafeHandle` to absolutely ensure it is disposed properly.

Sorry I didn't spot this in my last review!

Reply:
Unless it's extremely short lived, in which case it can be handled with try/finally everywhere it's used. But a `SafeHandle` is probably easier and safer.